MetaMask/metamask-extension
# UX Multichain: Removed top border from activity list item

This PR is to remove the top border from the Activity list item in order to have the same UI without the grey underline for all three tabs in Home Page.

Fixes #18917 

## Screenshots/Screencaps 

### Before

![Screenshot 2023-05-02 at 9 19 05 PM](https://user-images.githubusercontent.com/39872794/235717987-dd5521f4-627b-42ca-9182-0c1501077861.jpg)


### After

![Screenshot 2023-05-02 at 9 17 40 PM](https://user-images.githubusercontent.com/39872794/235717645-4e13d2c9-7298-4b43-bbdd-e13ae30049c0.jpg)


## Manual Testing Steps

- Go to the Activity Tab on the home screen
- Check there is no grey underline below any Tab heading


## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
